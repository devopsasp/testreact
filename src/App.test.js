import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer'
import App from './App';

test('snap shot testing', () => {
 
    const component=renderer.create(<App/>)
    let tree=component.toJSON()
    expect(tree).toMatchSnapshot()
  
});
