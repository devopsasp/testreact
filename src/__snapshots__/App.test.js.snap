// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snap shot testing 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <img
      alt="logo"
      className="App-logo"
      src="logo.svg"
    />
    <p>
      Edit 
      <code>
        src/App.js
      </code>
       and save to reload.
    </p>
    <a
      className="App-link"
      href="https://reactjs.org"
      rel="noopener noreferrer"
      target="_blank"
    >
      React Test Demo
    </a>
  </header>
</div>
`;
